docs/translation.md

```diff
@@ -0,0 +1,37 @@
+# Translation Guidelines
+
+Thank you for your interest in contributing translations to Keycloak! Translations play a crucial role in making Keycloak accessible to users from diverse linguistic backgrounds. This document provides guidelines for contributing translations and getting started with Weblate.
+
+## Why Translation Matters
+Translation enables Keycloak to reach a wider audience by making the platform usable for speakers of various languages. By translating Keycloak, you help improve accessibility and usability for users who may not be proficient in English.
+
+## What to Translate
+We welcome translations for various aspects of Keycloak, including user interfaces, documentation, and error messages. Your contributions can greatly enhance the user experience for non-English speakers.
+
+## Getting Started with Weblate
+We use [Weblate](https://hosted.weblate.org/projects/keycloak/), a web-based translation platform, to manage translations for Keycloak. Here's how you can get started:
+
+- Sign Up: Visit Weblate to create an account.
+- Configuration: After signing up, configure your Weblate account settings according to your preferences. Please note that the committer email is set to the login email address by default. You can adjust this in your Weblate profile under Account settings.
+- Joining the Translation Team: Join the Keycloak translation team on Weblate to start contributing translations.
+
+### Guidelines and Conventions
+To ensure consistency and quality across translations, please adhere to the following guidelines:
+
+- Use formal or informal language as appropriate for the context.
+- Maintain consistency with existing translations.
+- Translate text accurately, ensuring that the meaning is preserved.
+- Translations will be committed to github.com/Keycloak after they were inserted on Weblate.org(Update every hour or so). An Administrator for that language has the ability to approve or decline that translation.
+### Currently, we are actively supporting translations in the following languages:
+
+- English
+- German
+- Dutch
+
+Please note that while we aspire to support more languages in the future, our current focus is on these three languages. We rely on volunteers like you to take the initiative in steering the translation efforts for additional languages. If you're interested in contributing translations for a language not listed above, we encourage you to register on Weblate and reach out to join the community to kickstart the translation process for your language.
+
+If you have any questions or need assistance, feel free to reach out to the language coordinators listed below:
```

Comment:
I think rather than 'coordinators' we should call them 'maintainers' of languages, where those contributing could just be named 'contributors'

Reply:
will do in next commit